MLS.Agent/Properties/launchSettings.json

```diff
@@ -10,31 +10,29 @@
   "profiles": {
     "IIS Express": {
       "commandName": "IISExpress",
-      "commandLineArgs": "hosted",
-      "launchBrowser": true,
+      "commandLineArgs": "--port 4242 hosted",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
-    "MLS.Agent": {
+    "Try": {
```

Comment:
Can you add this file to .gitignore? I'm tired of seeing it in PRs.

Reply:
I changed it intentionally in this PR to pass the port option